docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md

```diff
@@ -36,8 +36,6 @@ Ethernet port can be obtained on Ubuntu 22.04.
 1. Power on the DUT.
 1. Boot into the system.
 1. Log into the system by using the proper login and password.
-1. Connect docking station to power source.
```

Comment:
@Pokisiekk If we don't want to write about connecting the docking station during the test, please add information about needed connections in the section `Test setup`.

Reply:
Information is in the test setup